docs-src/content/using-cubic-sdk/installation.md

```diff
@@ -35,3 +35,17 @@ dotnet add package Cubic-SDK
 
 ### Android
 Building for Android requires more steps, so it is described on the [Android Integrations](https://cobaltspeech.github.io/sdk-cubic/using-cubic-sdk/android/) page.
+
+### iOS
+
+#### Swift Package Manager
+
+The [Swift Package Manager](https://swift.org/package-manager/) is a tool for automating the distribution of Swift code and is integrated into the `swift` compiler.
+
+Once you have your Swift package set up, adding swift-cubic as a dependency is as easy as adding it to the `dependencies` value of your `Package.swift`.
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/cobaltspeech/sdk-cubic.git", .upToNextMajor(from: "1.3.0"))
```

Comment:
Will this string need to be changed in the release-notes section of the readme? Is there a way to always get the latest tagged version (not master)